packages/forms-web-app/src/dynamic-forms/dynamic-components/list-add-more/index.njk

```diff
@@ -13,7 +13,7 @@
     <div class="{{question.width}}">
       {# TODO update to persist main question header text #}
       
-      <p class="govuk-heading-l">{{question.question}}</p>
+      <p class="govuk-heading-l">{{question.pageTitle}}</p>
```

Comment:
Looking at this, should this be an h1 and isPageHeading be set to false in the field set.
Or is it this way round on purpose?